meta-lmp-bsp/dynamic-layers/xilinx-tools/recipes-bsp/fpga-manager-script/files/0001-Correct-used-paths.patch

```diff
@@ -0,0 +1,160 @@
+From 84394ab52b5eedc5c1220b7988a44db617b98691 Mon Sep 17 00:00:00 2001
+From: Tim Anderson <tim.anderson@foundries.io>
+Date: Fri, 22 Jan 2021 22:56:30 +0000
+Subject: [PATCH] Correct used paths
+
+This updates the utility to use paths that are correct for this
+distribution.
+
```

Comment:
While the configpath change can easily land upstream, please add the following line (to track the patch upstream status):

Upstream-Status: Inappropriate [lmp specific]